bin/ci/build-and-run-tests.sh

```diff
@@ -45,6 +45,13 @@ echo "--- :key: Generating fake origin key"
 # we won't have access to any valid signing keys.
 hab origin key generate "$HAB_ORIGIN"
 
+echo "--- Installing the studio" 
+# Work around https://github.com/habitat-sh/habitat/issues/7219
+# This will ensure the correct version of the studio for the `hab` 
+# on our path is installed, and provide some informational output
+# about what version we intend to use. 
+hab studio version
```

Comment:
Does running this command effect some workaround? (We should file an issue because the docs entry https://www.habitat.sh/docs/habitat-cli/#hab-studio-version is empty)

Reply:
Nothing inherent to the version subcommand itself.  The particular bug is triggered when the studio is installed as part of the execution, so I needed a command that would force the install of the correct studio before we use it and didn't have side effects. `hab studio rm` or parsing `hab --version` and installing would have also worked, but `version` has the benefit of some potentially useful information when troubleshooting.